script.js

```diff
@@ -23,5 +15,18 @@ closeBtn.addEventListener('click', () => {
     overlay.style.display = "none";
 })
 
+BurgerEl.addEventListener("focus",() =>{
+    BurgerEl.addEventListener("keydown",(e)=>{
+        if(e.code == "Enter"){overlay.style.display = "block";}
```

Comment:
Could the `e` in "Enter" ever be lowercase in some browser? Because then the string match wouldn't evaluate to true? Maybe compare them in lowercase (`e.code.toLowerCase() === "enter"`), if this is a potential risk?